ProjectTemplate/Modules/Login/Tests/LoginViewModel_Tests.swift

```diff
@@ -0,0 +1 @@
+
```

Comment:
❓ are we okay with keeping the `LoginViewModel` tests empty?

Reply:
I think so, this isn't a guide how to test the code, but how the template could look like. Even though I solved that with a comment inside those empty test files.